heic2jpg.sh

```diff
@@ -2,15 +2,22 @@
 
 src=$1
 size=$(find $src -type f | grep -i '\.heic$' | wc -l)
-cnt=0
+iter=0
 
 for i in $(find $src -type f | grep -i '\.heic$')
 do
-	((cnt=cnt+1))
+	iter=$((iter+1))
 	target=$(echo $i | sed 's/heic$/jpg/i')
-	printf "$i convert to $target\n"
+	# https://www.tutorialkart.com/bash-shell-scripting/bash-date-format-options-examples/
+	FILE_DATE=$(/bin/date +%Y%m%d%H%M.%S -r ${target})
```

Comment:
This format looks like this `202001100851.47`, which is for `2020-01-10 08:51:47`, a format accepted by `touch` to set the new date. This doesn't take the Time Zone into account, but at least the converted file will be much closer to the original creation date than the time of creation of the converted file.